src/index.ts

```diff
@@ -76,6 +76,13 @@ export async function getOrientation (
   return info;
 }
 
+export async function setOrientation (
+  input: File | Buffer | ArrayBuffer,
+  orientation: number,
```

Comment:
It would be better to have this input type `OrientationCode | IOrientationInfo`

Reply:
I swapped to `OrientationCode` for now. I think I'll need a new test for to drive myself into adding `IOrientationInfo`.

Reply:
Sorry I now think `IOrientationInfo` is a little bit too much. Only `OrientationCode`  would be great.

Reply:
I was thinking the same thing